src/Keboola/StorageApi/Components.php

```diff
@@ -25,6 +25,14 @@ public function __construct(Client $client)
         $this->client = $client;
     }
 
+    /**
+     * @return Client
+     */
+    public function getClient()
```

Comment:
Abych se dostal ke klientovi z Components, musel jsem udelat **public** metodu. Vadi to necemu?

Reply:
IMHO je to špatný směr. Z components by sis klienta tahat neměl.

Reply:
Jo, zruseno.